src/Components/GEOSldas_GridComp/GEOSlandassim_GridComp/clsm_ensupd_read_obs.F90

```diff
@@ -4584,6 +4585,534 @@ subroutine read_obs_SMOS( date_time, N_catd, this_obs_param,      &
     
   end subroutine read_obs_SMOS
 
+
+  ! *****************************************************************
+  
+  subroutine read_MODISscf_hdf( N_files, date_time, N_data, fnames, &
+       MODIS_lon, MODIS_lat, MODIS_SCF)
+    
+    ! Purpose: read preprocessed (renamed) snow cover data from daily MODIS Terra MOD10C1, version 6.1 (https://nsidc.org/data/mod10c1/versions/61) 
+    !   - Data currently located at /discover/nobackup/projects/S2SHMA/MODIS/MOD10C1_V61/ (2010-2022)
+    !   - Daily dataset with spatial resolution of 0.05 deg on CMG grid, missing days 2016 d. 50-58
+    ! 
+    ! Procedures:
+    !   - dtstep_assim and ref_time is restricted to 3 hr and 00z, respectively
+    !   - Generate the latitude and longitude on CMG grid
+    !   - Select longitude band considering MODIS observation time (local time: ~10:30am)     ! HARDWIRED OVERPASS TIME, WHAT ABOUT AQUA !?!?!!?!?!?!?!
+    !
+    ! Table. Longitude band based on the assimilation time step (in UTC) 
+    !-----------------------------------------------------------------------------
+    ! UTC | 0000    | 0300   | 0600  | 0900 | 1200 | 1500   | 1800    | 2100    
```

Comment:
There is an error in how the longitude bands are constructed such that the edges are included in two assimilation cycles. I am trying to identify where this happens, but havent quite yet.
<img width="912" alt="image" src="https://github.com/GEOS-ESM/GEOSldas/assets/53305329/97110b3c-9b9c-4bc9-9aad-04ac51ca81a1">

Reply:
Thanks, @lcandre2.  I don't think this is worth pursuing right now.  I will rewrite the reader eventually.  I had hoped to get it done before the shutdown, and there's still a sliver of a chance.  If it doesn't happen before Monday, it'll happen after the shutdown.